### DISCUSSION TITLE
Is a newline in `displayName` expected behavior?

### DISCUSSION THREAD

**Discussion by kawarimidoll** [bluesky-social/atproto#1418] [Dev Debugging] [answered]
We can add newline characters (`\n`) in `displayName` by `putRecord()` API.
Is this expected behavior?
I'm sure that Bluesky and Atproto doesn't need to have exactly the same style as other services, but this is a little curious (and funny).

screenshot↓
![image](https://github.com/bluesky-social/atproto/assets/8146876/3daf08e7-513f-427f-b34d-e9cfe3695a96)

**Comment by pfrazee** [comment, marked as answer]
Heh the client ought to be sanitizing this. Thanks for the heads up.
